case-server/src/main/java/com/powsybl/caseserver/CaseService.java

```diff
@@ -66,6 +68,9 @@
     @Lazy
     private CaseInfosService caseInfosService;
 
+    @Value("#{${max-public-cases:systemProperties['maxPublicCases']?: '-1'}}")
```

Comment:
why `:systemProperties['maxPublicCases']?` ?

also it's better to  just put the default value -1 in the shipped application.yml than here in the code ?